### PR TITLE
Adds NER Pipeline and Species Entity HTML API endpoint

### LINKED ISSUE #57: Create spaCy Pipeline Module
Currently we are creating the spaCy 3.x Pipeline using a forked branch of [spacy-lookup](https://github.com/sul-dlss-labs/spacy-lookup) in the data-prep.ipynb notebook. This pipeline should be moved into it's own module in `src/lib/` so that the FastAPI and the tests can use this pipeline.

Blocks #58

### PR DESCRIPTION
**Pull request by jermnelson**
Fixes #57 

Added directions to wiki for adding the `paper_tei` volume on the EC2 VM.

~~This PR requires an environment variable `PAPERS_TEI` to point to a directory (not in source control) containing the TEI-XML files. I will upload a tar file from the Google Drive to AWS EC2 instance and extract them to a directory parallel with spoc and manually set the variable in the VM.~~ Mounted the papers as Docker volume instead.

### CODE REVIEW

File: .github/workflows/spoc-app.yml [hunk 1]
```diff
@@ -33,5 +33,7 @@ jobs:
       run: black --check src --exclude spacy-lookup
     - name: Type checking with mypy
       run: mypy src --exclude spacy-lookup
+    - name: Download spaCy English Medium
```
Comment (jgreben) [inline .github/workflows/spoc-app.yml:36]: If this is a dependency as opposed to a lint or test, maybe move it up next to the Install Dependencies section?

**Review by jgreben — COMMENTED** [review]
Is the ner pipeline pretty much just implementing the factory software, and does not need any explicit testing here?

File: .github/workflows/spoc-app.yml (hunk 1 above)
Comment (jermnelson) [reply .github/workflows/spoc-app.yml]: Done in latest commit

**Comment by jermnelson** [comment]
> Is the ner pipeline pretty much just implementing the factory software, and does not need any explicit testing here?

Good suggestion, added new tests for the pipeline since we're actually doing some custom entity matching based on species, habitats, and location.

**Review by jgreben — APPROVED** [review]

**Pull request merged**